### ISSUE TITLE
Hope to increase the Chinese lunar calendar

### ISSUE THREAD

**Issue by netforfreedom** [persian-calendar/persian-calendar#916]
Hope to increase the Chinese lunar calendar.thank you very much

**Comment by ebraminio** [comment]
Definitely, this has been on my radar https://github.com/persian-calendar/persian-calendar/issues/824 and you can see even I'm already showing Chinese year name in Astronomy section of the app just I couldn't provide a Chinese calendar implementation for myself yet https://github.com/persian-calendar/calendar/blob/main/src/main/java/io/github/persiancalendar/calendar/ChineseDate.java I ported one from a PHP code but it didn't work as expected yet.
